examples/dt/uni-alpha/control-plane/service-values.yaml

```diff
@@ -0,0 +1,138 @@
+---
+apiVersion: v1
+kind: ConfigMap
+
+metadata:
+  name: service-values
+  annotations:
+    config.kubernetes.io/local-config: "true"
+
+data:
+  cinderVolumes:
```

Comment:
cinderVolumes accepts list of backends, so you need to insert an additional YAML tag for each backend. See https://github.com/openstack-k8s-operators/cinder-operator/blob/main/config/samples/backends/lvm/iscsi/backend.yaml#L18 (the "lvm-iscsi" tag). Then, L12..L22 of this file should be indented under the "lvm-iscsi" layer.

Also note you need to add a nodeSelector that ensures the pod is scheduled on the same "openstack.org/cinder-lvm=" node where the LVM backend data is stored.

Reply:
Okay, hope the rendered output addresses the requirement

```
  cinder:
    apiOverride:
      route: {}
    template:
      cinderAPI:
        override:
          service:
            internal:
              metadata:
                annotations:
                  metallb.universe.tf/address-pool: internalapi
                  metallb.universe.tf/allow-shared-ip: internalapi
                  metallb.universe.tf/loadBalancerIPs: 172.17.0.80
              spec:
                type: LoadBalancer
      cinderBackup:
        customServiceConfig: |
          [DEFAULT]
          backup_driver = cinder.backup.drivers.swift
        networkAttachments:
        - storage
        replicas: 1
      cinderScheduler:
        replicas: 1
      cinderVolumes:
        lvm-iscsi:
          customServiceConfig: |
            [DEFAULT]
            enabled_backends = lvm

            [lvm]
            image_volume_cache_enabled = false
            volume_driver = cinder.volume.drivers.lvm.LVMVolumeDriver
            volume_group = cinder-volumes
            target_protocol = iscsi
            target_helper = lioadm
            volume_backend_name = lvm_iscsi
            target_ip_address=172.18.0.5
            target_secondary_ip_addresses = 172.19.0.5
          nodeSelector:
            openstack.org/cinder-lvm: ""
      databaseInstance: openstack
      secret: osp-secret
```

Reply:
This is looking good!! But please consider:
- While replicas will default to 1, it would be better if the CR explicitly set replicas: 1
- It's missing the networkAttachments (which should be just the storage network, similar to cinder-backup's networkAttachments)
- The 'enabled_backends' setting is actually optional (the cinder-operator will auto-populate it). It would be good to test this, so I recommend removing these lines:

```
[DEFAULT]
enabled_backends = lvm
```

Reply:
> This is looking good!! But please consider:
>     * It's missing the networkAttachments (which should be just the storage network, similar to cinder-backup's networkAttachments)

Sorry, ignore this ^^ comment. As you already note in one of the YAML files, cinder's LVM backend does not require networkAttachments.